Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -147,6 +148,14 @@ void GeneralPane::CreateBasic()
   m_checkbox_cheats = new ConfigBool(tr("Enable Cheats"), Config::MAIN_ENABLE_CHEATS);
   basic_group_layout->addWidget(m_checkbox_cheats);
 
+  m_checkbox_load_games_into_memory =
+      new ConfigBool(tr("Load Games Into Memory"), Config::MAIN_LOAD_GAMES_INTO_MEMORY);
+  basic_group_layout->addWidget(m_checkbox_load_games_into_memory);
+  m_checkbox_load_games_into_memory->SetDescription(
+      tr("Load active discs and WAD files into memory in the background."
```

Comment:
What "active" and what "discs" means is maybe not clear to everyone. How about "Loads the running game into memory in the background"? (I know this is technically inaccurate in the case of running a title that's installed on the NAND, but everyone should hopefully have their NAND on a fast local disk, and it's one of the rarest ways of running games too.)